pkgs/tools/networking/inetutils/default.nix

```diff
@@ -1,18 +1,18 @@
 { stdenv, lib, fetchurl, ncurses, perl, help2man }:
 
 stdenv.mkDerivation rec {
-  name = "inetutils-1.9.4";
+  name = "inetutils-2.0";
 
   src = fetchurl {
     url = "mirror://gnu/inetutils/${name}.tar.gz";
```

Comment:
Is it worth pulling the xz variant of the tarball instead?

Reply:
It's ~1M smaller, switched to it.